### ISSUE TITLE
Backup task takes over 2 hrs and it doesn't stop. Normally it takes 10 min.

### ISSUE THREAD

**Issue by Krzysztonek** [sabeechen/hassio-google-drive-backup#572]
Since today morning, the backup task takes over 2 hrs and it doesn't stop. Normally it takes 10 min.
I'm not able to stop this process, even after rebooting my HA machine.
Please, see the below log:

 Addon version: `0.105.2`
 Home Assistant Version: `2022.3.5`
 Supervisor Version: `2022.03.4`
 Supervisor Channel: `stable`
 Hassos Version: `7.5`
 Docker Version: `20.10.9`
 Architecture: `amd64`
 Machine: `generic-x86-64`
 Date: `2022-03-17T14:09:28.551736+00:00`
 Timezone: `Europe/Warsaw`
 Failure Time: `Never`
 Last Good Sync: `2022-03-17T13:13:10.158927+00:00`
 ###### Exception:
 ```
 No error could be identified automatically.
 ```
`
 Backups:
 ```
 {
    "HomeAssistant": {
        "backups": 6,
        "retained": 0,
        "deletable": 6,
        "name": "HomeAssistant",
        "title": "Home Assistant",
        "latest": "2022-03-17T12:13:07Z",
        "max": 5,
        "enabled": true,
        "icon": "home-assistant",
        "ignored": 0,
        "size": "2.8 GB",
        "ignored_size": "0.0 B",
        "free_space": "192.8 GB"
    },
    "GoogleDrive": {
        "backups": 11,
        "retained": 1,
        "deletable": 10,
        "name": "GoogleDrive",
        "title": "Google Drive",
        "latest": "2022-03-17T01:30:01Z",
        "max": 10,
        "enabled": true,
        "icon": "google-drive",
        "ignored": 0,
        "size": "6.4 GB",
        "ignored_size": "0.0 B",
        "free_space": "4.6 GB"
    }
}
 ```
 ###### Config:
 ```
 {
    "Setting.MAX_BACKUPS_IN_HA": 5,
    "Setting.MAX_BACKUPS_IN_GOOGLE_DRIVE": 10,
    "Setting.DAYS_BETWEEN_BACKUPS": 1.0,
    "Setting.IGNORE_OTHER_BACKUPS": true,
    "Setting.IGNORE_UPGRADE_BACKUPS": true,
    "Setting.DELETE_BEFORE_NEW_BACKUP": true,
    "Setting.BACKUP_NAME": "REDACTED",
    "Setting.BACKUP_TIME_OF_DAY": "02:30",
    "Setting.BACKUP_PASSWORD": "REDACTED",
    "Setting.GENERATIONAL_DAYS": 6,
    "Setting.GENERATIONAL_WEEKS": 10,
    "Setting.EXCLUDE_FOLDERS": "media",
    "Setting.USE_SSL": true,
    "Setting.SEND_ERROR_REPORTS": true,
    "Setting.BACKGROUND_COLOR": "#000000"
}
 ```
 ###### Addon Logs:
 ```
 03-17 12:43:55 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/about?fields=storageQuota
03-17 12:43:55 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271t5y8TtjrwGxw5CwW2aHkJQMGsl_bYhCY%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%2CdriveId%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
03-17 13:13:07 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/addons/self/info
03-17 13:13:07 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/info
03-17 13:13:08 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/core/info
03-17 13:13:08 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/supervisor/info
03-17 13:13:08 INFO [backup.ha.hasource] Requesting a new backup
03-17 13:13:08 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/backups/new/partial
03-17 13:13:08 WARNING [backup.ha.hasource] A backup was already in progress
03-17 13:13:08 ERROR [backup.ui.uiserver] Error serving POST http://home-assistant:8123/backup?custom_name=stary+hoymiles&retain_drive=false&retain_ha=false
03-17 13:13:08 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:701 (error_middleware)
  addon/backup/ui/uiserver.py:267 (backup)
  addon/backup/model/coordinator.py:236 (startBackup)
  addon/backup/model/coordinator.py:319 (_withSoftLock)
  addon/backup/model/coordinator.py:241 (_startBackup)
  addon/backup/ha/hasource.py:214 (create)
  addon/backup/ha/hasource.py:98 (raiseIfNeeded)
backup.exceptions.exceptions.BackupInProgress

03-17 13:13:08 DEBUG [backup.model.syncer] Sync requested by HomeAssistant
03-17 13:13:08 INFO [backup.model.coordinator] Syncing Backups
03-17 13:13:08 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/supervisor/info
03-17 13:13:08 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/backups
03-17 13:13:08 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/1t5y8TtjrwGxw5CwW2aHkJQMGsl_bYhCY/?fields=id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%2CdriveId&supportsAllDrives=true
03-17 13:13:08 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/about?fields=storageQuota
03-17 13:13:09 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271t5y8TtjrwGxw5CwW2aHkJQMGsl_bYhCY%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%2CdriveId%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
03-17 14:13:10 DEBUG [backup.model.syncer] Sync requested by Coordinator
03-17 14:13:10 INFO [backup.model.coordinator] Syncing Backups
03-17 14:13:10 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/supervisor/info
03-17 14:13:10 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/backups
03-17 14:13:10 DEBUG [backup.drive.driverequests] Requesting refreshed Google Drive credentials
03-17 14:13:10 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/1t5y8TtjrwGxw5CwW2aHkJQMGsl_bYhCY/?fields=id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%2CdriveId&supportsAllDrives=true
03-17 14:13:11 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/about?fields=storageQuota
03-17 14:13:11 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271t5y8TtjrwGxw5CwW2aHkJQMGsl_bYhCY%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%2CdriveId%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
03-17 15:09:23 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/addons/self/info
03-17 15:09:23 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/info
03-17 15:09:23 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/core/info
03-17 15:09:23 DEBUG [backup.ha.harequests] Making Hassio request: http://supervisor/supervisor/info
 ```
 ###### Supervisor Logs:
 ```
 [32m22-03-17 14:13:10 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup[0m
[32m22-03-17 14:13:10 INFO (MainThread) [supervisor.api.middleware.security] /backups access from cebe7a76_hassio_google_drive_backup[0m
[32m22-03-17 14:43:07 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[33m22-03-17 14:56:12 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[33m22-03-17 14:56:12 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[33m22-03-17 14:56:12 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[32m22-03-17 15:01:57 INFO (SyncWorker_3) [supervisor.docker.interface] Restarting ghcr.io/home-assistant/generic-x86-64-homeassistant[0m
[32m22-03-17 15:01:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Received message 8:1000 is not str[0m
[32m22-03-17 15:01:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed[0m
[32m22-03-17 15:01:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Received message 8:1000 is not str[0m
[32m22-03-17 15:01:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed[0m
[32m22-03-17 15:02:29 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready[0m
[32m22-03-17 15:02:48 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.FREEZE[0m
[32m22-03-17 15:02:48 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete[0m
[32m22-03-17 15:03:06 INFO (MainThread) [supervisor.auth] Auth request from 'core_mosquitto' for 'mqtt'[0m
[32m22-03-17 15:03:06 INFO (MainThread) [supervisor.auth] Home Assistant not running, checking cache[0m
[33m22-03-17 15:03:32 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[33m22-03-17 15:03:32 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[33m22-03-17 15:03:32 WARNING (MainThread) [supervisor.addons.options] Option 'require_ssl' does not exist in the schema for Node-RED (a0d7b954_nodered)[0m
[32m22-03-17 15:09:16 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance[0m
[32m22-03-17 15:09:21 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize[0m
[32m22-03-17 15:09:21 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered[0m
[32m22-03-17 15:09:22 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize[0m
[32m22-03-17 15:09:22 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered[0m
[32m22-03-17 15:09:23 INFO (MainThread) [supervisor.api.middleware.security] /core/info access from cebe7a76_hassio_google_drive_backup[0m
[32m22-03-17 15:09:23 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup[0m
[32m22-03-17 15:09:23 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running[0m
[32m22-03-17 15:09:23 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running[0m
[32m22-03-17 15:09:28 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m

 ```
 ###### Home Assistant Core Logs:
 ```
 [33m2022-03-17 15:09:16 WARNING (MainThread) [homeassistant.components.media_player] Updating samsungtv media_player took longer than the scheduled update interval 0:00:10[0m
[31m2022-03-17 15:09:22 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/discovery_flow.py", line 74, in _async_process_pending_flows
    await gather_with_concurrency(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 192, in gather_with_concurrency
    return await gather(
  File "/usr/src/homeassistant/homeassistant/util/async_.py", line 190, in sem_task
    return await task
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 203, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 230, in _async_init
    result = await self._async_handle_step(flow, flow.init_step, data, init_done)
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 325, in _async_handle_step
    result: FlowResult = await getattr(flow, method)(user_input)
  File "/usr/src/homeassistant/homeassistant/components/yamaha_musiccast/config_flow.py", line 89, in async_step_ssdp
    if not await MusicCastDevice.check_yamaha_ssdp(
  File "/usr/local/lib/python3.9/site-packages/aiomusiccast/musiccast_device.py", line 87, in check_yamaha_ssdp
    res = await client.get(location)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client.py", line 559, in _request
    await resp.start(conn)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/client_reqrep.py", line 898, in start
    message, payload = await protocol.read()  # type: ignore[union-attr]
  File "/usr/local/lib/python3.9/site-packages/aiohttp/streams.py", line 616, in read
    await self._waiter
aiohttp.client_exceptions.ServerDisconnectedError: Server disconnected[0m
cat: can't open '/sys/class/hwmon/hwmon0/temp1_input': No such file or directory
[31m2022-03-17 15:09:23 ERROR (SyncWorker_7) [homeassistant.components.command_line] Command failed: cat /sys/class/hwmon/hwmon0/temp1_input[0m
[33m2022-03-17 15:09:27 WARNING (MainThread) [homeassistant.components.media_player] Updating aquostv media_player took longer than the scheduled update interval 0:00:10[0m

 `

**Comment by sabeechen** [comment]
The supervisor logs don't go far back enough to tell whats going on, but here are a couple possibilities:

- Something might have happened to make your backups suddenly a lot bigger.  I've seen this happen to some people when they add a sensor that constantly logs information and fills up the database, or if they accidentally drop a large file into the config folder.  you might want to check your /config folder in Home assistant to see it it suddenly has any large files.
- The supervisor might just be stuck.  It has bugs sometimes.

Please also note that the only way to stop a backup is to restart the host machine or restart the supervisor.  Restarting just Home Assistant or the addon won't do it.  This is a limitation in Home Assistant's supervisor.  You should be able to stop the backup by going to <kbd>Configuraiton</kbd> > <kbd>Add-ons Backups & Supervisor</kbd> > <kbd>System</kbd> > Either <kbd>Restart Supervisor</kbd> or <kbd>Reboot Host</kbd>.  Alternatively you can just power-cycle the machine, I would recommend a full reboot.  Its impossible for a backup to "resume" after a reboot, and the add-on will wait 10 minutes after a system restart before it attempts to create a new backup.  I suspect a full reboot will resolve this issue.

**Comment by Krzysztonek** [comment]
Thank you, @sabeechen 👍🏻 
I'll come back with the result once my investigation is done.
